frontend/src/views/Layout/ContentFooter.vue

```diff
@@ -15,6 +15,13 @@
           <a href="https://github.com/gradido/gradido/releases/latest" target="_blank">
             App version {{ version }}
           </a>
+          <a
+            v-if="hash"
+            :href="'https://github.com/gradido/gradido/commit/' + hash"
+            target="_blank"
+          >
+            ({{ shortHash }})
```

Comment:
Can you please make a test for this block in the ContentFooter.spec.js 

The test should resemble to the one done for "App version {{ version }}"
With a test if hash filled and one with hash empty.